google/cloud/bigtable/test_proxy/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A minimal CMakeList.txt showing how to use the Cloud Bigtable test proxy
+# in CMake-based projects.
+
+cmake_minimum_required(VERSION 3.10...3.24)
```

Comment:
I would expect this is part of the top-level project and therefore this is not required.

Reply:
Removed.